styles/style.css

```diff
@@ -122,6 +160,9 @@ p {
 .hero {
   background-image: url("header-bike.jpg");
```

Comment:
This image link is still broken; currently it is looking for the image in `/styles/header-bike.jpg`, however the image is in the `/images` folder. There are two ways you can fix this link; either by _relative_ import ("what is the path to get to the destination from where I am currently") or _absolute_ ("starting at the very top level, what is the path to the image").

Reply:
This goes for the other images on the page too; although they are in the HTML using the `src` attribute, the same logic applies.